packages/promise_jsoo/promise_jsoo.0.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Js_of_ocaml bindings to JS Promises with supplemental functions"
+maintainer: ["Max Lantas <mnxn.dev@outlook.com>"]
+authors: ["Max Lantas <mnxn.dev@outlook.com>"]
+license: "MIT"
+homepage: "https://github.com/mnxn/promise_jsoo"
+bug-reports: "https://github.com/mnxn/promise_jsoo/issues"
+depends: [
+  "dune" {>= "2.7"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
```
From the tests it seems to support only ocaml versions more recent than 4.08

Reply:
Can you upstream this change?

Reply:
Added the ocaml version constraint.

Why was the dune dependency dropped?

Reply:
My  mistake sorry, I meant to append!